### DISCUSSION TITLE
dApp review and approval

### DISCUSSION THREAD

**Discussion by uchibeke** [WalletConnect/walletconnect-monorepo#2151] [Developer Support]
Hi, I created a project and it's still pending review

Could it be reviewed and approved, please? This is a blocker for launching our dApp on Celo to support Valora

**UID:** cadeeacb-1c80-464a-bc17-055a044c7572

Thanks

**Comment by finessevanes** [comment]
Hey - I actually can't find your dapp with the UID. Getting approved on the Explorer doesn't impact WalletConnect usage.
